cli/api/daemon.go

```diff
@@ -709,7 +709,7 @@ func (d *daemon) startAgent() error {
 		// register the API
 		glog.V(0).Infoln("registering ControlPlaneAgent service")
 		if err = d.rpcServer.RegisterName("ControlPlaneAgent", hostAgent); err != nil {
-			glog.Fatalf("could not register ControlPlaneAgent RPC server: %v", err)
+			glog.Fatalf("could not register RPC server named ControlPlaneAgent: %v", err)
```

Comment:
We shouldn't be using the name "ControlPlane" if we can avoid it.  Please change to control center